changelog.dd

```diff
@@ -10,11 +10,12 @@ $(LI $(RELATIVE_LINK2 asm-attributes, Asm statements can now be used in pure not
 
 $(BUGSTITLE Library Changes,
 $(LI $(RELATIVE_LINK2 volatile-load-store, $(D volatileLoad) and $(D volatileStore) intrinsics were added.))
+$(LI $(RELATIVE_LINK2 gc-options, $(B Experimental:) The garbage collector can now be configured.))
 $(LI $(RELATIVE_LINK2 aa-keyvalue, $(D byKeyValue) was added.))
 
 $(BUGSTITLE Language Changes,
 
-$(LI $(LEGACY_LNAME2 asm_attributes, asm-attributes, Asm statements can now be used in pure nothrow @nogc @trusted code.:)
+$(LI $(LNAME2 asm-attributes, Asm statements can now be used in pure nothrow @nogc @trusted code.:)
```

Comment:
could you please put $(D ) around code-related stuff?
    
    $(LI $(LNAME2 asm-attributes, $(D asm) statements can now be used in $(D pure), $(D nothrow), $(D @nogc), or $(D @trusted) code.:)